tools/ci_build/github/linux/docker/scripts/training/ortmodule/stage1/requirements_torch1.11.0_rocm5.2/requirements.txt

```diff
@@ -0,0 +1,12 @@
+# transformers requires sklearn
```

Comment:
is this file useful? or just a placeholder?
last time when I tested ortmodule, the requirements (uninstall+install) failed quite a few UTs. Thus I think wget and pytorch_lightning==1.6.0 would be good enough. pytorch_lightning is part of stage2, but I think we can put these two together in stage1. see bottom of orttraining-pai-ci-pipeline.yml for ref.

Reply:
but you can leave it as is, till we revisit it

Reply:
I think there are two things.
1. Because of we remove some test from packaging pipeline, some packages in this file are no longer needed.  And we need to make sure which package in this file need to be deleted.
2. This file only needed by Rocm packaging pipeline, our Rocm CI pipeline use a already built image which built by this Dockerfile : https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/github/pai/rocm-ci-pipeline-env.Dockerfile. 
I think the reason for using a built container is to save time and ensure the environment is stable. If you want to install other package, I think we can add it on this Dockerfile and rebuild the docker image. I can add wget and pytorch_lightning==1.6.0 when we update our CI pipeline to Rocm5.2+toch1.11.